src/FakeItEasy/Core/DelegateRaiser.cs

```diff
@@ -68,7 +68,7 @@ private static void AssertThatValuesSatisfyCallSignature(MethodInfo callMethod,
 
             throw new FakeConfigurationException(
                 "The event has the signature ({0}), but the provided arguments have types ({1})."
-                    .FormatInvariant(fakeSignature, actionSignature, "Raise"));
+                    .FormatInvariant(fakeSignature, actionSignature, nameof(Raise)));

```

Comment:
In retrospect, this one may be an error. It refers to the class `Raise`. I will restore.